AB_Hands_On/AB_Hands_On.ipynb

```diff
@@ -0,0 +1,6511 @@
+{
```

Comment:
<p>The posterior numbers are not the same in the plot above. I know theres randomness, I hestitant to set a seed. Perhaps just dnot write numbers down in the text, that way they wont conflict with whatever ends up happening during sampling</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>It's always around 70% and 30% chance, so I used this wording in the text</p>